src/components/Badge/__snapshots__/Badge.test.ts.snap

```diff
@@ -0,0 +1,113 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Badge /> Default story renders snapshot 1`] = `
+<span
+  class="badge__wrapper"
+>
+  <div
+    class="badge badge--empty"
+  />
+  <div
+    class="fpo flex items-center justify-center h-8 w-8"
+  >
+    Ava
+  </div>
+</span>
+`;
+
+exports[`<Badge /> IconBadge story renders snapshot 1`] = `
+<span
+  class="badge__wrapper"
+>
+  <div
+    class="badge"
+  >
+    <svg
+      aria-hidden="true"
+      class="icon"
+      fill="currentColor"
+      height="1rem"
+      style="--icon-size: 1rem;"
+      width="1rem"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <use
+        xlink:href="test-file-stub#alarm"
+      />
+    </svg>
+  </div>
+  <div
+    class="fpo flex items-center justify-center h-8 w-8"
+  >
+    Ava
+  </div>
+</span>
+`;
+
+exports[`<Badge /> LargeBadgeableObject story renders snapshot 1`] = `
+<span
+  class="badge__wrapper"
+>
+  <div
+    class="badge"
+  >
+    10
```

Comment:
I'm wondering what we should do about screen reader support here. "10" on its own doesn't seem like it would be very meaningful without the context of the badge appearance. We know when we see that styling that the number refers to some kind of update or notification, but what's the right way to communicate that to a screen reader and what API should the component use to encourage that?

Reply:
@dierat this is a good question. A good approach here is that we provide a facility to `<Badge>` for labeling it with text that captures the meaning of the thing being badged AND the count or meaning of the badge itself. The badge is hidden, so screen readers don't simply say "5". That number usually has a "type" as well. 

Rough example:

```
<Badge badge={5} description="You have 5 unread notifications"><NotificationsComponent /></Badge>
```

Reply:
Since the badge itself is `aria-hidden` the object it's attached to needs to have an accessible name for screen reader support. In the stories, I added `aria-label`s but not sure how I can further enforce besides documentation? @dierat @booc0mtaco

Reply:
I was thinking of something similar to @booc0mtaco 's example above where a prop is passed into the badge components, which we can then type as a required prop.

Reply:
As discussed, using a prop would mean having to dig down into the Badgeable object and applying the prop as a description or sr-only text but would run into various complications regarding Fragments, already existing descriptions, which element to attach to, etc, so going forward with strong documentation about needing an accessible name acknowledging the badge